### ISSUE TITLE
WebUI: monitor: architecture selection

### ISSUE THREAD

**Issue by matwey** [openSUSE/open-build-service#4952]
# Issue Description

At page https://build.opensuse.org/monitor the user may select different architectures at the bottom of the page. However, when the user reload page the following inconsistency happens.
Architecture select box points to previously selected value, but the plots are from x86_64.

Expected Result
============
Architecture select box and plots are always in consistent state.

How to Reproduce
===============
1. Go to https://build.opensuse.org/monitor

![screenshot_20180506_152242](https://user-images.githubusercontent.com/156627/39673274-dc9a2c26-5141-11e8-8cc7-db7e5d8e4f61.png)

2. Click on 'Architecture' and select `armv7l'

![screenshot_20180506_152313](https://user-images.githubusercontent.com/156627/39673279-e9d48fee-5141-11e8-981d-8540b3c1084c.png)

3. Click 'reload' button in your browser of choice. At least Firefox keeps previously selected value at the select box.

![screenshot_20180506_152333](https://user-images.githubusercontent.com/156627/39673288-03a2f758-5142-11e8-803d-2f61b9a74567.png)

**Comment by ChrisBr** [comment]
Only seems to happen in firefox. Chrome seems to behave fine.

**Comment by coolo** [comment]
yeah, firefox upstream just ignores that it breaks *every* ajax page :(

**Comment by coolo** [comment]
https://laravel.io/forum/06-16-2016-forms-inputs-value-are-not-reset-after-browser-refresh suggests an easy workaround

**Comment by frispete** [comment]
Does anybody have a related mozilla bugzilla number at hand?
